src/create_links_sinks.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+import argparse
+import boto3
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+
+# Set up parser to accept CLI flags
+parser = argparse.ArgumentParser(add_help=True)
+parser.add_argument('--organization', help="AWS Organization", type=str, required=True)
+parser.add_argument('--organization-unit', help="AWS Organization Unit", type=str, required=True)
+parser.add_argument('--sink-name', help="Human-friendly AWS Metric Sink Name, defaults to \"Centralized-Monitoring\"", type=str, required=False, default="Centralized-Monitoring")
+parser.add_argument('--profile', help="AWS CLI Config Profile Name for the Management Account, uses your default AWS profile if not specified", type=str, required=False, default=None)
+parser.add_argument('--regions', help="Double quoted, comma separated list of regions to create a sink and StackSet in. If not specified, a Sink/StackSet will be created in all default AWS regions", type=str, required=False)
+parser.add_argument('--excluded-accounts', help="[Optional] double quoted, comma separated list of accounts to exclude (ex. --exclude \"12345678910, 09876543210\"). Must exclude suspended accounts", type=str, required=False, default=None)
+
+# AWS Default regions
+aws_default_regions = ["us-east-2", "us-east-1", "us-west-1", "us-west-2",
+		"ap-south-1", "ap-northeast-3", "ap-northeast-2", "ap-southeast-1", "ap-southeast-2",
+		"ap-northeast-1", "ca-central-1", "eu-central-1", "eu-west-1", "eu-west-2", "eu-west-3",
+		"eu-north-1", "sa-east-1"]
+
+policy = """{"Version":"2012-10-17","Statement":[{"Effect":"Allow","Principal":"*","Action":["oam:CreateLink","oam:UpdateLink"],"Resource":"*","Condition":{"ForAllValues:StringEquals":{"oam:ResourceTypes":"AWS::CloudWatch::Metric"},"ForAnyValue:StringEquals":{"aws:PrincipalOrgID":"%s"}}}]}"""
```

Comment:
Could we pull this out to a prettified JSON template file similar to `link-cfn-template.yaml`? Just a personal preference for ease of reading/editing

Reply:
I'd have to dig up the documentation, but there's some AWS specificities at play here. I had it in a json file originally, but the api didn't like that

Reply:
https://docs.aws.amazon.com/cli/latest/reference/oam/put-sink-policy.html#options
`The policy must be in JSON string format with quotation marks escaped and no newlines.`

Reply:
Ah got it; the no-newlines is odd